src/sentry/snuba/metrics/query_builder.py

```diff
@@ -426,16 +438,38 @@ def _build_where(self) -> List[Union[BooleanCondition, Condition]]:
 
     def _build_groupby(self) -> List[Column]:
         groupby_cols = []
-        for field in self._metrics_query.groupby or []:
+
+        def generate_aliased_expression(column: Column, alias: str) -> AliasedExpression:
```

Comment:
not sure that we really need this 🤔

Reply:
What's wrong with doing this? Just cleaner imo

Reply:
Nothing bad, just asking why do we need 1 line function, I'm not sure that it's cleaner cause it doesn't incapsulate any complexity, just using function instead of class.

Reply:
Actually I agree with you.. will change